### PR TITLE
Added landing page section-2 and 4

### PR DESCRIPTION
**Pull request by Yazhinivenkatesh**
This PR has the following
-> Added landing page sections 2 and 4
-> Design changes in header and footer

### CODE REVIEW

File: frontend/src/assets/images/imageAssets.ts [hunk 1]
```diff
@@ -1,3 +1,8 @@
 export { default as CurveXLogo } from "../images/MainLogo.png";
 export { default as FooterLogo } from "../images/FooterLogo.svg";
 export { default as WalletIcon } from "../images/WalletIcon.png";
+export { default as Feature1 } from "./feature-1.png";
+export { default as Feature2 } from "./feature-2.png";
+export { default as Feature3 } from "./feature3.png";
+export { default as Feature4 } from "./feature-4.png";
+export { default as Greek } from "./Greek.png";
```
Comment (nehaannvinu) [inline frontend/src/assets/images/imageAssets.ts:8]: Kindly provide meaningful names to assets for clarity

File: frontend/src/assets/styles/material/_ant_button.scss [hunk 2]
```diff
@@ -1,11 +1,12 @@
...
 
 .ant-btn {
-    width: 150px;
-    height: 42px;
-    border-radius: 10px;
+    width: 170px;
+    font-size: 14px;
+    height: 48px;
+    padding: 10px;
+    border-radius: 12px;
```
Comment (nehaannvinu) [inline frontend/src/assets/styles/material/_ant_button.scss:8]: Kindly make custom changes in specific scss file and not in global styles.

File: frontend/src/app/components/Header/Header.tsx [hunk 3]
```diff
@@ -4,15 +4,21 @@ import { Link, useLocation } from "react-router-dom";
...
 const Header = () => {
+  const [account, setAccount] = useState('');
   const location = useLocation();
 
   const { metaState, connectWallet } = useMetamaskProvider();
 
+  useEffect(() => {
+    setAccount(metaState.account[0]);
+  }, [metaState.account])
+
```
Comment (nehaannvinu) [inline frontend/src/app/components/Header/Header.tsx:18]: Kindly avoid fixing bugs that are discussed & fixed. May result in conflicts for others

**Review by nandhabn — CHANGES_REQUESTED** [review]

File: frontend/.gitignore [hunk 4]
```diff
@@ -4,6 +4,7 @@
 /node_modules
 /.pnp
 .pnp.js
+package-lock.json
```
[the remark this answers is not in the document]
Comment (Yazhinivenkatesh) [reply frontend/.gitignore]: okay

**Review by nandhabn — APPROVED** [review]

**Pull request merged**